pwndbg/wrappers/__init__.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pwndbg.which
+
+
+class OnlyWithCommand(object):
+    def __init__(self, func):
+        self.func = func
```

Comment:
This is Python, most of the objects can be freely changed and so you can add an attribute to a function object:
```python
def foo(): pass
foo.bar = "123"
```
So you can do `function.command_path = ...` in your example.

I am also not sure about fetching the command name based on file name - it is not that obvious(?) and "ugly" for programs with e.g. space in their name.